src/app.js

```diff
@@ -90,29 +91,80 @@ function printScore() {
         } else {
             cell.innerHTML = score + " <img src='src/img/green_check.png' width='25'>";
         }
+        students[i].score = score; // update student score
     }
-}
+};
 
 /* Clear table and add new rows depending on the input number */
 
 function clearAndCompleteTable() {
     for (let i = 1; i < table.rows.length;) {
         table.deleteRow(i);
     }
+    students = []; // clear array
     for (let i = 1; i <= numberOfStudentsInput.value; i++) {
         addRowToStudentsTable(i);
     }
-}
+};
 
 /* Add rows to students table depending on the input number */
 
 function addRowToStudentsTable(i) {
     let row = table.insertRow(i);
     let cell1 = row.insertCell(0);
     let cell2 = row.insertCell(1);
-    let cell3 = row.insertCell(2);
-    let cell4 = row.insertCell(3);
+    row.insertCell(2);
+    // TODO: until proper task to be done
+    // row.insertCell(3);
 
     cell1.innerHTML = i; // Number
     cell2.innerHTML = "Elvis Presley";
-}
\ No newline at end of file
+    let student = new Student(i, "Elvis Presley");
+    students.push(student);
+};
+
+/* Send button*/
+
+let sendButton = document.getElementById("sendButton");
+
+/* Send button is disabled by default */
+
+sendButton.disabled = true;
+
+/* Send button is enabled after clicking on evaluate buttons */
+
+evaluateButton1.addEventListener("click", function () {
+    sendButton.disabled = false;
+});
+evaluateButton2.addEventListener("click", function () {
```

Comment:
You have defined two "click" event listener. You could inline in one function

Reply:
Done